tests/Queries/PercolateQueryTest.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Spatie\ElasticsearchQueryBuilder\Tests\Queries;
+
+use Exception;
+use PHPUnit\Framework\TestCase;
+use Spatie\ElasticsearchQueryBuilder\Queries\PercolateQuery;
+
+final class PercolateQueryTest extends TestCase
```

Comment:
```suggestion
class PercolateQueryTest extends TestCase
```

We don't use final in our packages